### ISSUE TITLE
Meta-issue: Discordification of Matrix/Synapse/Dendrites (feature request)

### ISSUE THREAD

**Issue by DonaldTsang** [vector-im/element-meta#963]
### Description

There are at least a dozen issues relating to the design of Riot being too user unfriendly, and that people are suggesting designs similar to established apps like Discord and Slack.
(Please post related issues in this page for further discussion)

### Issue Types:

* Better Communities (AKA Discord "servers")
* Seamless E2E with better UX
* Better "unread messages" with ping as primary type
* Tag-based user and mod/admins types (similar to Discord)
* Mumble as a default widget (instead of Jitsi as it is bad on mobile)
* (others not included in the list above)

---

I have been a Riot user for about one and a half years, and I would hope for these features to give people a better user experience (features recommended from friends in Discord)
- [ ] Room feature support
    - [ ] Custom server-based and room-based emojis
       - Have custom server emojis downloaded and be usable as a basic feature
    - [ ] Emoji collection sharing
        - Similar to Telegram, Messenger, LINE and Whatsapp, but shareable and if possible, self-hosted or user-hosted
    - [x] Emoji reaction for texts and files (useful for voting)
    - [x] Automated (non-manual) key sharing
        - Some people are sick of needing to sharing key every time someone gets in the room, they would like to have auto key sharing once in an E2E room
    - [x] Multi-room search
- [ ] Community support
    - [x] Community id generation
    - [ ] Community room mass generation
    - [x] Community room searching
    - [x] Community multi-admin multi-mod support 
    - [ ] Community and sub-community sort
- [ ] Role and access support
    - [ ] Mass room invitation by role or community
        - invitation to a server once application is approved
    - [x] Mass room banning/kicking by role or community
        - for when "unfavorable" behavior by certain users happen
    - [x] Community room sub-categorization and access
        - for visitor vs club member vs club staff separation
    - [ ] Role application and removal
        - for upgrading users to automatically bring them to certain rooms
    - [ ] Announcement rooms by role
        - rooms that allows viewing but not posting unless it is staff
    - [ ] Reaction allowance by role
       - for voting by certain role but not others
- [x] audio chat support
    - [ ] self-host SIP room support (tying SIP with server)
    - [ ] self-host  Mumble room support (tying Mumble with server)
    - [ ] Radio/Playlist bot support
- [ ] collaborative support
    - [ ] self-host Etherpad/Firepad support (tying *pad with server)
    - [ ] self-host Hackmd support (tying Hackmd with server)

Cross-reference: https://github.com/matrix-org/synapse/issues/4030

**Comment by t3chguy** [comment]
Broken transfer, duplicate of https://github.com/vector-im/element-meta/issues/1476
